src/entries/popup/hooks/swap/useSwapAssets.ts

```diff
@@ -94,42 +94,48 @@ export const useSwapAssets = () => {
     searchQuery: debouncedAssetToBuyFilter,
   });
 
-  const assetAddressesToFetchPrices = useMemo(() => {
-    const assetAddressesFromSearch = searchAssetsToBuySections
-      .map((section) => section.data?.map((asset) => asset.address) || [])
-      .flat();
-
-    const assetToBuyAddress = (assetToBuy?.address ||
-      prevAssetToBuy?.address) as Address;
-    if (
-      assetToBuyAddress &&
-      !assetAddressesFromSearch.includes(assetToBuyAddress)
-    ) {
-      assetAddressesFromSearch.push(assetToBuyAddress);
-    }
-    return assetAddressesFromSearch;
-  }, [assetToBuy, prevAssetToBuy, searchAssetsToBuySections]);
-
-  const { data: assetsWithPrice = [] } = useAssets({
-    assetAddresses: assetAddressesToFetchPrices,
+  // const assetAddressesToFetchPrices = useMemo(() => {
+  //   const assetAddressesFromSearch = searchAssetsToBuySections
+  //     .map((section) => section.data?.map((asset) => asset.address) || [])
+  //     .flat();
+
+  //   const assetToBuyAddress = (assetToBuy?.address ||
+  //     prevAssetToBuy?.address) as Address;
+  //   if (
+  //     assetToBuyAddress &&
+  //     !assetAddressesFromSearch.includes(assetToBuyAddress)
+  //   ) {
+  //     assetAddressesFromSearch.push(assetToBuyAddress);
+  //   }
+  //   return assetAddressesFromSearch;
+  // }, [assetToBuy, prevAssetToBuy, searchAssetsToBuySections]);
```

Comment:
are we leaving it commented?

Reply:
sorry, i did this locally and forgot to push